Scoop/Controller/TripExtraDetailsViewController.swift

```diff
@@ -0,0 +1,182 @@
+    
+    private let dateLabel = UILabel()
+    private let dateField = UITextField()
+    
+    private let timeLabel = UILabel()
+    private let timeField = UITextField()
+    
+    private let detailsLabel = UILabel()
+    private let detailsField = UITextField()
+
+
+    override func viewDidLoad() {
+        
+
+        
+        self.navigationItem.title = "Trip Details"
+        view.backgroundColor = .white
+        
+        travelersLabel.text = "Number of Travelers"
+        travelersLabel.textColor = .black
+        view.addSubview(travelersLabel)
+        
+        let peopleImage = UIImage(systemName: "person.2")
+        let peopleView = UIImageView(image: peopleImage)
+        peopleView.tintColor = .black
+        view.addSubview(peopleView)
+        
+        count1Field.placeholder = "0"
+        count1Field.textColor = .black
+        view.addSubview(count1Field)
+        
+        toLabel.text = "to"
+        toLabel.textColor = .black
+        view.addSubview(toLabel)
+        
+        count2Field.placeholder = "0"
+        count2Field.tintColor = .black
+        view.addSubview(count2Field)
+        
+        
+        dateLabel.text = "Date of Trip"
+        dateLabel.textColor = .black
+        view.addSubview(dateLabel)
+        
+        let calendarView = UIImageView(image: UIImage(systemName: "calendar"))
+        calendarView.tintColor = .black
+        view.addSubview(calendarView)
+        
+        dateField.placeholder = "mm/dd/yyyy"
+        dateField.textColor = .black
+        view.addSubview(dateField)
+        
+        
+        timeLabel.text = "Time of Trip"
+        timeLabel.textColor = .black
+        view.addSubview(timeLabel)
+        
+        let clockView = UIImageView(image: UIImage(systemName: "clock"))
+        clockView.tintColor = .black
+        view.addSubview(clockView)
+        
+        timeField.placeholder = "1:33 AM"
+        timeField.textColor = .black
+        view.addSubview(timeField)
+        
+        
+        detailsLabel.text = "Other Details"
+        detailsLabel.textColor = .black
+        view.addSubview(detailsLabel)
+        
+        let writeView = UIImageView(image: UIImage(systemName: "square.and.pencil"))
+        writeView.tintColor = .black
+        view.addSubview(writeView)
+        
+        detailsField.placeholder = "enter details..."
+        detailsField.textColor = .black
+        view.addSubview(detailsField)
+
+//        let transImage = UIImage(systemName: "location")
+//        let transView = UIImageView(image: transImage)
+//        transView.tintColor = .black
+//        transView.frame = CGRect(x: 0, y: 0, width: 20, height: 20)
+//        view.addSubview(transView)
+        
+        
+        travelersLabel.snp.makeConstraints { make in
+            make.top.equalToSuperview().offset(300)
+            make.leading.equalToSuperview().offset(80)
+        }
+        
+        peopleView.snp.makeConstraints { make in
```

Comment:
Remember to reorganize these constraints so that you set the constraint for a view immediately after you add it to as subview